core/src/main/java/io/questdb/cairo/frm/file/ContinuousFileColumnPool.java

```diff
@@ -0,0 +1,160 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cairo.frm.file;
+
+import io.questdb.cairo.ColumnType;
+import io.questdb.cairo.frm.FrameColumn;
+import io.questdb.cairo.frm.FrameColumnPool;
+import io.questdb.cairo.frm.FrameColumnTypePool;
+import io.questdb.std.FilesFacade;
+import io.questdb.std.ObjList;
+import io.questdb.std.str.Path;
+
+import java.io.Closeable;
+
+public class ContinuousFileColumnPool implements FrameColumnPool, Closeable {
```

Comment:
nit: in other similar cases we use the term "contiguous", not "continuous".

Reply:
done